examples/games/rock_paper_scissors/play_rock_paper_scissors.qnt

```diff
@@ -0,0 +1,57 @@
+// -*- mode: Bluespec; -*-
+module rock_paper_scissors {
+    // Constants for player names
+    const PLAYER1 : str
+    const PLAYER2 : str
+    // Enum types for the possible moves and game status
```

Comment:
These are technically sum types (although there's a pretty blurred line between them, we call it sum types in Quint, so it's good to be consistent in the examples)
```suggestion
    // Sum types for the possible moves and game status
```